zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -103,6 +111,9 @@
   // ZkEventThread. Otherwise the retry request might block the normal event processing.
   protected final ZkAsyncRetryThread _asyncCallRetryThread;
 
+  private final boolean _syncOnNewSession;
+  final String _syncPath = new String("/");
```

Comment:
private static?

Reply:
Directly do _syncPath = "/"; shall help to avoid unnecessary object creating.

Reply:
Could you please also make it static? And follow our naming convention, SYNC_PATH.